Homework_8_2/Shape.java

```diff
@@ -0,0 +1,42 @@
+package Homework_8_2;
+import java.util.Objects;
+
+public abstract class Shape {
```

Comment:
This is not a fully used abstract. Please create some abstract methods(For ex. calculateArea, calculatePerimeter)